src/components/WorkExperienceForm/WorkExperienceForm.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import {
+  ProForm,
+  ProFormDateRangePicker,
+  ProFormGroup,
+  ProFormList,
+  ProFormText,
+  ProFormTextArea
+} from '@ant-design/pro-components';
+import { WorkExperienceType } from '../CvViewer/CvViewer.types';
+import { useAppDispatch } from '../../hooks/redux';
+import { addWorkExperience, removeWorkExperience, updateWorkExperience } from '../../features/cv/cvSlice';
+
+type WorkExperienceFormProps = {
+  workExperience: WorkExperienceType[];
+};
+
+const emptyWorkExperience: WorkExperienceType = {
+  id: '0',
+  companyName: 'Company name',
+  position: 'Position on the job',
+  remark: ''
+};
+
+export const WorkExperienceForm: React.FC<WorkExperienceFormProps> = (props) => {
+  const dispatch = useAppDispatch();
+  const actionGuard = {
+    beforeAddRow: (defaultValue, insertIndex) => {
+      // eslint-disable-next-line no-param-reassign
+      defaultValue.companyName = 'Company name';
+      // eslint-disable-next-line no-param-reassign
+      defaultValue.position = 'Position on the job';
+      dispatch(addWorkExperience({ ...emptyWorkExperience, id: insertIndex.toString() }));
+      return true;
+    },
+    beforeRemoveRow: async (index) => {
+      dispatch(removeWorkExperience(index.toString()));
+      return true;
+    }
+  };
+
+  return (
+    <ProForm
+      onValuesChange={(changeValues) => {
+        const index = changeValues.workExperience.findIndex((item) => !!item);
+        console.log('changeValues index', index);
+        dispatch(updateWorkExperience({ id: index.toString(), ...changeValues.workExperience[index] }));
+      }}
+      submitter={{
+        resetButtonProps: {
+          style: {
+            display: 'none'
+          }
+        },
+        submitButtonProps: {
+          style: {
+            display: 'none'
+          }
+        }
+      }}
+    >
+      <ProFormList
```

Comment:
Will we have here select that will modify answers received from an AI?

Reply:
Because it seems like you've deleted and haven't moved it

Reply:
We need to add it with the logic of modifying answers, but this should be connected with redux